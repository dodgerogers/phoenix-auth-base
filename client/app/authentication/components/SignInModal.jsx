import React from 'react'
import PropTypes from 'prop-types';
import AuthenticationModal from './AuthenticationModal';
import SignInFormContainer from '../containers/SignInFormContainer';
import AuthenticationLinks from './AuthenticationLinks';


const SignInModal = (props) => {
  return (
    <AuthenticationModal title="Login" {...props}>
      <SignInFormContainer {...props} />
      <AuthenticationLinks />
    </AuthenticationModal>
  );
}

export default SignInModal;
