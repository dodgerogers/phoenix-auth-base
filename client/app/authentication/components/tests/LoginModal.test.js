import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SignInModal from '../SignInModal';


describe('SignInModal', () => {
  const fullProps = {
    close: jest.fn(),
  };

  const pureComponent = (props) => shallow(
    <SignInModal {...props} />
  );

  it('matches snapshot', () => {
    const wrapper = pureComponent(fullProps);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
