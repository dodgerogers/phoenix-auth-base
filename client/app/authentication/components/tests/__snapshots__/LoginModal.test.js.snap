// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginModal matches snapshot 1`] = `
<AuthenticationModal
  close={[Function]}
  title="Login"
>
  <Connect(ReduxForm)
    close={[Function]}
  />
  <AuthenticationLinks />
</AuthenticationModal>
`;
